main.go

```diff
@@ -33,7 +33,7 @@ func calculateQueueLatency(contents string) float64 {
 	}
 
 	if enqueuedAt, exists := job["enqueued_at"]; exists {
-		latency := float64(time.Now().UnixNano())/1000000.0 - enqueuedAt.(float64)
+		latency := float64(time.Now().UnixNano())/1000000000.0 - enqueuedAt.(float64)
```

Comment:
ナノ秒に合わせて桁数を増やしていただいたと思うのですが、すいません、よく見ると[前回の PR に載せた](https://github.com/feedforce/datadog-sidekiq/pull/27#discussion_r1524126391)実際の `enqueued_at` の値である `1679980269.5128906` というタイムスタンプはナノ秒ではなく、マイクロ秒でした 💦 

ですので、以下のコードが正しそうです。

```suggestion
		latency := float64(time.Now().UnixMicro())/1000000.0 - enqueuedAt.(float64)